models/coamps_nest/README.rst

```diff
@@ -0,0 +1,38 @@
+##################
+COAMPS Nest README
+##################
+
+Contents
+========
+
+#. `Overview`_
+#. `Terms of Use`_
+
+Overview
+========
+
+An updated version of the COAMPS model interfaces and scripts.
```

Comment:
These interfaces are Lanai-only ... we have a development version that may be Manhattan-ized, but that is not what is in this repository.

Reply:
Sounds good. This is just a conversion of what was in the documentation. If we need to update, rather than convert the documentation, I think that's a good thing to put in a new issue and to make a separate pull request.